### DISCUSSION TITLE
крашится при запуске discord.bat

### DISCUSSION THREAD

**Discussion by rugk0** [Flowseal/zapret-discord-youtube#518] [Q&A] [answered]
уже все пробывал запустить не открывается все равно,от имени администратора тоже
![image](https://github.com/user-attachments/assets/ae4dd370-5bc2-4a1b-ab3f-1aa72ee7e72c)

**Comment by Vargavinter** [comment, marked as answer]
Ты из архива пытаешься запустить? Разархивируй в папку на рабочий стол.
